src/navigation/MerchantPayoutStack/index.tsx

```diff
@@ -40,12 +41,12 @@ const MerchantPayoutStack: FunctionComponent<NavigationInjectedProps> & {
 
   const { authCredentials } = useContext(AuthStoreContext);
   if (!authCredentials[key]) {
-    throw new Error("No auth credentials found");
+    throw new Error(i18n.t("thrownErrors.noAuthCred"));
   }
 
   const { allCampaignConfigs } = useContext(CampaignConfigsStoreContext);
   if (!allCampaignConfigs[key]) {
-    throw new Error("No campaign config found");
+    throw new Error(i18n.t("thrownErrors.noCampaignConfig"));
```

Comment:
@sebastianquek Noticed these are for merchant flow, and in the normal default flow we don't throw these errors. Not very sure about data flow -- will these be shown to users?

Reply:
Good catch, the flow should match what's in the normal default flow. I must have missed this out in the other PR.
@dhemutton could you help to update these to match the normal default flow?